data/pritamrajput.json

```diff
@@ -21,4 +21,6 @@
       "icon": "FaGithub"
     }
   ]
```

Comment:
Please add a missing comma `,` here to validate your json! 

```suggestion
  ],
```

Reply:
Okay and than create new pull request ?

Reply:
No-no, you did it! 🎉